packages/resource-deployment/templates/on-demand-url-scan-schedule.template.json

```diff
@@ -25,7 +25,7 @@
             "userIdentity": {
                 "autoUser": {
                     "scope": "task",
-                    "elevationLevel": "nonadmin"
+                    "elevationLevel": "admin"
```

Comment:
why we need to switch to admin?

Reply:
With nonadmin the task fails with an error "you need admin access to run this task"